.github/workflows/publish.yml

```diff
@@ -7,8 +7,6 @@ on:
       - main
       - release-*
     tags: ["v*"]
-  release:
-    types: [published]
```

Comment:
As far as I can see, this is triggering the release twice. First from the tag and then when publishing the release.
The latter can be seen in action here https://github.com/akka/alpakka/actions/runs/2999580924